Documentation/development-environment.md

```diff
@@ -9,6 +9,7 @@ indent: true
 * [Using KVM/QEMU and Kubespray](#using-kvmqemu-and-kubespray)
 * [Using VirtualBox and k8s-vagrant-multi-node](#using-virtualbox-and-k8s-vagrant-multi-node)
 * [Using Vagrant on Linux with libvirt](#using-vagrant-on-linux-with-libvirt)
```

Comment:
@BlaineEXE @galexrt @leseb Is this document still something to maintain? Setting up a K8s cluster is really outside the scope of Rook.

Reply:
It is about development and hopefully helping out new devs, so I think it's still relevant.